jenkins/mergerequest

```diff
@@ -6,23 +6,14 @@ pipeline {
     }
 
     stages {
-        stage('generate testdata') {
+        stage('Pytest') {
             steps {
-                sh './jenkins/scripts/step_generate_testdata.sh'
-            }
-        }
-
-        stage('run unittests amip') {
-            steps {
-                sh './jenkins/scripts/step_execute_unittest_amip.sh'
-            }
-        }
-
-        stage('run unittests mch') {
-            steps {
-                sh './jenkins/scripts/step_execute_unittest_mch.sh'
+                sh'''
+                source /project/g110/icon/probtest/conda/miniconda/bin/activate
```

Comment:
Yes this would be better. I wanted to add a coverage test as well, but I did not know how to change the probtest conda env.

There are quite some requirements around in probtest, which one is the right one?

Reply:
Yes, it is a bit redundant here. I would go for the conda that is also used in github actions.

Reply:
OK, fine by me. We could also switch to github actions completely. But let's leave it to the next PR.